### DISCUSSION TITLE
Design files of Tamagui themes?

### DISCUSSION THREAD

**Discussion by thegitparticle** [tamagui/tamagui#365] [General]
Where do I find design scheme or figma/sketch files for the themes to help design my app using tamagui themes?

**Comment by montnyc** [comment]
Found a version here: https://github.com/tamagui/tamagui/commit/5085e1932f05f4c5426ec1974a7534c3d15fc248#diff-e9403d211b6c5715659f460f071235513adee76a48b204b03b6cd8ca0b067b8a
